--- src/ffmpeg-5.0.1/libavfilter/metal/utils.m
+++ src/ffmpeg-5.0.1/libavfilter/metal/utils.m
@@ -31,7 +31,7 @@ void ff_metal_compute_encoder_dispatch(id<MTLDevice> device,
     BOOL fallback = YES;
     // MAC_OS_X_VERSION_10_15 is only defined on SDKs new enough to include its functionality (including iOS, tvOS, etc)
 #ifdef MAC_OS_X_VERSION_10_15
-    if (@available(macOS 10.15, iOS 11, tvOS 14.5, *)) {
+    if (@available(macOS 10.15, iOS 13, tvOS 14.5, *)) {
         if ([device supportsFamily:MTLGPUFamilyCommon3]) {
             MTLSize threadsPerGrid = MTLSizeMake(width, height, 1);
             [encoder dispatchThreads:threadsPerGrid threadsPerThreadgroup:threadsPerThreadgroup];

--- src/ffmpeg-5.0.1/libavcodec/videotoolbox.c
+++ src/ffmpeg-5.0.1/libavcodec/videotoolbox.c
@@ -1156,7 +1156,12 @@
 #endif
 #if HAVE_KCVPIXELFORMATTYPE_420YPCBCR10BIPLANARVIDEORANGE
     if (depth > 8) {
-        return AV_PIX_FMT_P010;
+        #if TARGET_OS_IPHONE
+             /* iOS doesn't support 10 bit textures in GLES */
+             return AV_PIX_FMT_BGRA;
+         #else
+             return AV_PIX_FMT_P010;
+         #endif
     }
 #endif
 
